packages/pixeloven-react/routing/src/types.ts

```diff
@@ -1,16 +1,15 @@
 import { ConnectedComponentClass } from "react-redux";
 import { match } from "react-router";
 import { RouteComponentProps, RouteProps } from "react-router-dom";
-import { Dispatch } from "redux";
 
 /**
  * @todo Should remove the reliance on redux make types more generic
  * @todo need to expose react router through this package
  */
-/* tslint:disable no-any */
+/* tslint:disable no-any ban-types */
```

Comment:
This should probably be tslint:disable-next-line: no-any and then another next-line comment for ban-types on L:12

Reply:
Good call @jimmynotjim ! 

I updated to use disable next line for `ban-types`, but I kept this disable no-any here because `any` is used a couple times in the file. I did add `/* tslint:enable no-any` at the end of the file to re-enable the rule.